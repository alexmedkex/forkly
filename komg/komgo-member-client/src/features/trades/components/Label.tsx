import * as React from 'react'
import styled from 'styled-components'

const Label = styled.span`
  display: inline-block;
  font-weight: bold;
  width: 200px;
  text-align: right;
  padding-right: 20px;
`

export default Label
