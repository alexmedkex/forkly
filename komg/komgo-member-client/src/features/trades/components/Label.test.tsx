import * as React from 'react'
import Label from './Label'
import * as renderer from 'react-test-renderer'

it('should match snapshot', () => {
  const tree = renderer.create(<Label>OK</Label>).toJSON()

  expect(tree).toMatchSnapshot()
})
