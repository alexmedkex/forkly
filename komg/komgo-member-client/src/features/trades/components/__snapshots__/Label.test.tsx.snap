// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
.c0 {
  display: inline-block;
  font-weight: bold;
  width: 200px;
  text-align: right;
  padding-right: 20px;
}

<span
  className="c0"
>
  OK
</span>
`;
